Src/Plugins/Security/SEC101_045.PostgreSqlConnectionStringValidator.cs

```diff
@@ -6,6 +6,7 @@
 using System.Text;
 
 using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.HelpersUtiliesAndExtensions;
```

Comment:
>HelpersUtiliesAndExtensions [](start = 67, length = 27)

remove this when u rename all the namespaces

Reply:
Removed.

---
In reply to: [575575696](https://github.com/microsoft/sarif-pattern-matcher/pull/189#discussion_r575575696) [](ancestors = 575575696)